src/EmployerPR/SFA.DAS.EmployerPR/Application/Queries/GetRoatpProviders/GetRoatpProvidersQueryHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MediatR;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EmployerPR.Application.Queries.GetRoatpProviders;
+public class GetRoatpProvidersQueryHandler : IRequestHandler<GetRoatpProvidersQuery, GetRoatpProvidersQueryResult>
```

Comment:
You can now use class level constructors 
`public class GetRoatpProvidersQueryHandler(IRoatpV2TrainingProviderService _roatpV2TrainingProviderService) : IRequestHandler<GetRoatpProvidersQuery, GetRoatpProvidersQueryResult>`

Reply:
I tried to implement this, and came back with message 'Class cannot have primary constructor'. Visual studio offered no suggestions, and google did not turn up any suggestions.

I think the code is pretty clear as is, so leaving this as is.

Reply:
I did find some examples and they seem to reflect what you say, but Visual Studio just doesn't work with it, so not changing.